src/sentry/static/sentry/app/actionCreators/guides.tsx

```diff
@@ -42,11 +43,18 @@ export function recordFinish(guide: string, orgId: string) {
       status: 'viewed',
     },
   });
+
+  const user = ConfigStore.get('user');
+  if (!user) {
+    return;
+  }
+
   const data = {
     eventKey: 'assistant.guide_finished',
     eventName: 'Assistant Guide Finished',
     guide,
     organization_id: orgId,
+    user_id: parseInt(user.id, 10),
```

Comment:
Can we update getsentry to handle coercing the field here https://github.com/getsentry/getsentry/blob/e1a11fff526636f1ec3705d67166958e8e734e6f/static/getsentry/gsApp/utils/trackAnalyticsEvent.jsx#L8

Reply:
will do